recipe/meta.yaml

```diff
@@ -27,7 +27,6 @@ requirements:
     - loguru ~=0.7
     - pydantic ~=1.9  # Definitely NOT compatible with pydantic 2.0 (!)
     - pandas ~=1.4.4
-    - seaice ~=6.0
```

Comment:
Removed `seaice` requirement. This was only used in the `compare` subpackage, extracted to `seaice_ecdr`.